coderd/parameters.go

```diff
@@ -114,6 +116,10 @@ func (api *API) templateVersionDynamicParameters(rw http.ResponseWriter, r *http
 		return
 	}
 
+	var staticDiagnostics hcl.Diagnostics
+	// If the err is sql.ErrNoRows, an empty terraform values struct is correct.
+	staticDiagnostics = staticDiagnostics.Extend(parameterProvisionerVersionDiagnostic(tf))
```

Comment:
I think this can simplify down a bit

```suggestion
	// If the err is sql.ErrNoRows, an empty terraform values struct is correct.
	staticDiagnostics := parameterProvisionerVersionDiagnostic(tf)
```